### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

https://github.com/crate-ci/typos

### CODE REVIEW

**Comment by szepeviktor** [comment]
Here is `.typos.toml`
```toml
[files]
extend-exclude = [
    ".git/",
    ".github/actions/elastic-beanstalk/bin/index.js",
    "client/packages/core/__tests__/src/data/zeneca/triples.json",
    "server/dev-resources/triples.csv",
    "server/resources/sample_triples/",
    # Strange plurals
    "client/packages/core/__tests__/src/instaql.test.js",
    "server/src/instant/db/datalog.clj",
]
ignore-hidden = false

[default]
extend-ignore-re = [
    "Kevin Lacker",
    "\\bC’mone\\b",
    "\\bOT rabbit\\b",
    "(pk_test|pk_live)_[0-9A-Za-z]{72,}",
    "playbackId: '[0-9A-Za-z]+'",
    'playbackId="[0-9A-Za-z]+"',
]

[default.extend-words]
edn = "edn"

[type.clojure.extend-words]
juxt = "juxt"

[default.extend-identifiers]
openpgp4fpr = "openpgp4fpr"

# Typos
"Github" = "GitHub"
```

File: client/sandbox/react-nextjs/pages/play/ephemeral.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ function Demo() {
   // END TYPE TESTS
 
   return (
-    <div className="flex fle items-center">
+    <div className="flex file items-center">
```
Comment (szepeviktor) [inline client/sandbox/react-nextjs/pages/play/ephemeral.tsx]: This needs attention ⚠️

**Review by stopachka — APPROVED** [review]
This is great, thank you!

**Review by stopachka — CHANGES_REQUESTED** [review]
(It looks like the clojure tests are failing. It may be transient. I will try to repro this locally when near a computer)

**Review by stopachka — APPROVED** [review]

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**